.hsds/data/hsdstest/home/.domain.json

```diff
@@ -0,0 +1 @@
+{"owner": "admin", "acls": {"admin": {"create": true, "read": true, "update": true, "delete": true, "readACL": true, "updateACL": true}, "default": {"create": false, "read": true, "update": false, "delete": false, "readACL": false, "updateACL": false}}, "created": 1708897646.0599918, "lastModified": 1708897646.0599918}
```

Comment:
It looks like the `hsds/data` folder was included by accident.

Reply:
removed